package.json

```diff
@@ -26,17 +26,19 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "ember-auto-import": "^1.6.0",
     "ember-cli-babel": "^7.22.1",
     "ember-cli-htmlbars": "^5.3.1",
-    "ember-modifier": "^2.1.1"
+    "ember-decorators-polyfill": "^1.1.5",
```

Comment:
Why is the decorators polyfill required? All current LTS versions of Ember should have native support for polyfills: [it landed in 3.10](https://blog.emberjs.com/2019/05/21/ember-3-10-released.html). Is there a specific app we are trying to support with this which isn't yet on 3.12+? If so, this seems fine, though we'll want to remember that dropping this in the future will constitute a breaking change b/c of the version support impact.

Reply:
Yes, ts-web is *still* on 3.8. :'(